$NetBSD: patch-ai,v 1.2 2009/12/09 09:23:51 he Exp $

Mirror a change committed just after the release: the open_pipe_for_writing
test should work everywhere.

--- t/op/io.t.orig	2009-10-20 23:30:03.000000000 +0200
+++ t/op/io.t
@@ -51,22 +51,6 @@ Tests various io opcodes.
     ok(1, 'open with null mode')
 .end
 
-.sub 'tt661_todo_test' :anon
-    # Checks whether the platform is linux, MSWin32, darwin: on other
-    # platforms, the following tests are todo'ed.
-    .include 'sysinfo.pasm'
-    $S0 = sysinfo .SYSINFO_PARROT_OS
-    if $S0 == 'linux' goto tt661_ok
-#    if $S0 == 'MSWin32' goto tt661_ok
-    if $S0 == 'darwin' goto tt661_ok
-    if $S0 == 'openbsd' goto tt661_ok
-
-    .return (0)
-
-  tt661_ok:
-    .return (1)
-.end
-
 .include 'iglobals.pasm'
 
 .sub 'open_pipe_for_reading'
@@ -103,8 +87,6 @@ Tests various io opcodes.
 .end
 
 .sub 'open_pipe_for_writing'
-    $I0 = tt661_todo_test()
-    unless $I0 goto open_pipe_for_writing_todoed
     .local pmc interp
     interp = getinterp
 
@@ -137,10 +119,6 @@ Tests various io opcodes.
   open_pipe_for_writing_failed:
     nok(1, 'open pipe for writing')
     .return ()
-
-  open_pipe_for_writing_todoed:
-    todo(1, 'Unimplemented in this platform, TT #661')
-
 .end
 
 # Local Variables:
